packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -140,6 +138,9 @@ const styles = StyleSheet.create({
         margin: 10,
         padding: 20,
     },
+    setWidth: {
+        width: 170,
```

Comment:
We should use `minWidth` in our examples to promote best practices around `i18n`.